dask_planner/src/parser.rs

```diff
@@ -319,41 +384,70 @@ impl<'a> DaskParser<'a> {
         }
     }
 
+    /// Parse a SQL PREDICT statement
+    pub fn parse_predict_model(&mut self) -> Result<DaskStatement, ParserError> {
+        // PREDICT(
+        //     MODEL model_name,
+        //     SQLStatement
+        // )
+        self.parser.expect_token(&Token::LParen)?;
+
+        // Expect `MODEL`
+        // This doesn't work? Somethign about Keyword in the equality comparison, therefore always fails.
```

Comment:
@andygrove when I tried to use `expect_token` it would fail with error saying `MODEL, not equal MODEL` I believe it was because the `Keyword` field was not set there? Either way would like to use this syntax it just wasn't working